### ISSUE TITLE
dynamically size the tech tree

### ISSUE THREAD

**Issue by 7yl4r** [7yl4r/techtreejs#19]
d3 canvas size should be set based on width/height of document/div. Possible by adding js methods to techtree.js or via css?

**Comment by mileswest07** [comment]
By "dynamically size" are you referring to [responsiveness](http://www.w3schools.com/css/css_responsive_intro.asp)? A media query would cover most CSS-related needs, but the SVG itself would need to be [redrawn whenever the browser resizes](http://www.w3schools.com/js/js_window.asp).

**Comment by 7yl4r** [comment]
Yes, that is what I meant. I've since done this on other projects; maybe I can pull some code over without much trouble.
